Mathlib/MeasureTheory/Group/Action.lean

```diff
@@ -103,6 +104,36 @@ theorem map_smul : map (c • ·) μ = μ :=
 
 end MeasurableSMul
 
+section SMulHomClass
+
+universe uM uN uα uβ
+variable {M : Type uM} {N : Type uN}  {α : Type uα} {β : Type uβ}
+  [MeasurableSpace M] [MeasurableSpace N] [MeasurableSpace α] [MeasurableSpace β]
+
+@[to_additive]
+theorem SMulInvariantMeasure_map [SMul M α] [SMul M β]
```

Comment:
This should be names either `SMulInvariantMeasure.map` or `smulInvariantMeasure_map`.